packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/azure/appidpinokta.md

```diff
@@ -13,3 +13,31 @@ Add the following endpoint URLs for the Azure AD IdP that you’re configuring.
 * **Authorization endpoint**: The URL of the Azure AD OAuth 2.0 authorization endpoint. For example: `https://login.microsoftonline.com/{Directory(tenant)ID}/oauth2/v2.0/authorize`
 * **Token endpoint**: The URL of the Azure AD token endpoint for obtaining access and ID tokens. For example: `https://login.microsoftonline.com/{Directory(tenant)ID}/oauth2/v2.0/token`
 * **JWKS endpoint**: The URL of the Azure AD JSON Web Key Set document. This document contains signing keys that are used to validate the signatures from the provider. For example: `https://login.microsoftonline.com/{Directory(tenant)ID}/discovery/v2.0/keys`
+
+In the optional **Authentication Settings** section:
+
+* **IdP Username**: This is the expression (written in Okta Expression Language) that is used to convert an IdP attribute to the app user's `username`. This IdP username is used for matching an app user to an Okta user.
+
+    For example, the value `idpuser.email` means that it takes the email attribute passed by the IdP and maps it to the Okta app user's `username` property.
+
+    You can enter an expression to reformat the value, if desired. For example, if the IdP username is `john.doe@mycompany.com`, then you could specify the replacement of `mycompany` with `endpointA.mycompany` to make the transformed username `john.doe@endpointA.mycompany.com`. See [Okta Expression Language](/docs/reference/okta-expression-language/).
+
+* **Filter > Only allow usernames that match defined RegEx Pattern**: Select this option to only authenticate users with transformed usernames that match a regular expression pattern in the text field that appears. This filters the IdP username to prevent the IdP from authenticating unintended users. Users are only authenticated if the transformed username matches the regular expression pattern.
```

Comment:
```suggestion
* **Filter > Only allow usernames that match defined RegEx Pattern**: Select this option to only authenticate users with transformed usernames that match a regular expression pattern in the text field that appears. This filters the IdP username to prevent the IdP from authenticating unintended users. 
```

Reply:
Suggest cutting this last sentence, seems like a duplicate of the main one